content/influxdb/cloud-iox/sql-reference.md

```diff
@@ -0,0 +1,614 @@
+
+## Literals
+
+A literal is an explicit value not represented by an identifier.
+
+### String literals
+
+String literals are surrounded by single quotes. 
+
+```sql
+'santa_monica'
+'pH'
+'average temperature'
+```
+
+### Numeric literals
+
+Number literals are positive or negative numbers that are either exact numbers or floats.
+
+```sql
+-- Integers
+10
++10
+-10
+
+-- Unsigned integers
+10::BIGINT UNSIGNED
++10::BIGINT UNSIGNED
+
+-- Floats
+10.78654
+-100.56
+```
+
+### Date and time literals
+
+The following date and time literals are supported:
+
+```sql
+'2022-01-31T06:30:30.123Z'     -- (RFC3339) 
+'2022-01-31T06:30:30.123'      -- (RFC3339-like)
+'2022-01-31 06:30:30.123'      -- (RFC3339-like)
+'2022-01-31 06:30:30'          -- ((RFC3339-like, no fractional seconds) 
+1643610630123000000::TIMESTAMP -- (Unix epoch nanosecond cast to a timestamp)
+```
+
+### Boolean literals
+
+Boolean literals are either `TRUE` or `FALSE`. 
+
+## Duration units
+
+Interval literals specify a length or unit of time. 
+
+```sql
+INTERVAL '4 minutes'
+INTERVAL '12 days 6 hours 30 minutes'
+```
+
+## Operators
+
+Operators are reserved words or characters which perform certain operations, inluding comparisons and arithmetic. 
+
+### Arithmetic operators
+
+Arithmetic operators take two numeric values (either literals or variables) and
+perform a calculation that returns a single numeric value.
+
+| Operator | Description    | Example | Result |
+| :------: | :------------- | ------- | -----: |
+|   `+`    | Addition       | `2 + 2` |    `4` |
+|   `-`    | Subtraction    | `4 - 2` |    `2` |
+|   `*`    | Multiplication | `2 * 3` |    `6` |
+|   `/`    | Division       | `6 / 3` |    `2` |
+|   `%`    | Modulo         | `7 % 2` |    `1` |
+
+### Comparison operators
+
+Comparison operators evaluate the relationship between the left and right operands and `TRUE` or `FALSE`.
+
+| Operator | Meaning                               | Example          |
+| :------: | :------------------------------------ | :--------------- |
+|   `=`    | Equal to                              | `123 = 123`      |
+|   `<>`   | Not equal to                          | `123 <> 456`     |
+|   `!=`   | Not equal to                          | `123 != 456`     |
+|   `>`    | Greater than                          | `3 > 2`          |
+|   `>=`   | Greater than or equal to              | `3 >= 2`         |
+|   `<`    | Less than                             | `1 < 2`          |
+|   `<=`   | Less than or equal to                 | `1 <= 2`         |
+|   `~`    | Matches a regular expression          | `'abc' ~ 'a.*'`  |
+|   `!~`   | Does not matches a regular expression | `'abc' !~ 'd.*'` |
+
+## Keywords
+
+The following reserved key words cannot be used as identifiers.
+
+```sql
+AND 
+ALL  
+AS  
+ASC  
+BOTTOM 
+CASE
+DESC 
+DISTINCT                                       
+EXISTS                                                      
+EXPLAIN  
+EXPLAIN ANALYZE   
```

Comment:
I think technically double words aren't keywords. So maybe the list should be more like:

```
EXPLAIN
ANALYZE
```

Rather than `EXPLAIN ANALYZE`

Reply:
fixed